extensions/amp-app-banner/1.0/test/validator-amp-app-banner.out

```diff
@@ -0,0 +1,38 @@
+FAIL
+|  <!--
+|    Test Description:
+|    Tests support for the amp-app-banner tag and required <head> metatdata.
+|  -->
+|  <!doctype html>
+|  <html ⚡ lang="en">
+|  <head>
+|    <meta charset="utf-8">
+|    <link rel="canonical" href="./regular-html-version.html">
+|    <meta name="viewport" content="width=device-width,minimum-scale=1">
+|    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|    <script async src="https://cdn.ampproject.org/v0.js"></script>
+|    <script async custom-element="amp-app-banner" src="https://cdn.ampproject.org/v0/amp-app-banner-1.0.js"></script>
+>>   ^~~~~~~~~
+amp-app-banner/1.0/test/validator-amp-app-banner.html:13:2 The extension 'amp-app-banner' is referenced at version '1.0' which is an invalid version. Please use a valid version of this extension. (see https://amp.dev/documentation/components/amp-app-banner)
```

Comment:
I posted this question in Slack, but I'll repeat:
This error makes sense, since `1.0` is clearly not published yet (since that's what this PR is) ... so is there any way to get this validator test to actually pass?  How did the other Bento components get it to pass?

Reply:
I talked with another developer. And he was told to hold off on the validator tests for a second PR after the component is published. So that might be the solution we need to go with. But I'm also asking our tech lead, and will get back to you if there is a better solution.